test/mean_function.jl

```diff
@@ -1,47 +1,53 @@
 @testset "mean_functions" begin
     @testset "ZeroMean" begin
-        P = 3
-        Q = 2
-        D = 4
-        # X = ColVecs(randn(rng, D, P))
-        x = randn(P)
-        x̄ = randn(P)
-        f = ZeroMean{Float64}()
+        rng, N, D = MersenneTwister(123456), 5, 3
```

Comment:
IMO it's more readable to have separate assignments (wonder if it's even discussed by BlueStyle):

```suggestion
        rng = MersenneTwister(123456)
        N = 5
        D = 3
```